mantle/cmd/kola/kola.go

```diff
@@ -173,6 +190,46 @@ func runRun(cmd *cobra.Command, args []string) error {
 		patterns = args
 	}
 
+	return kolaRunPatterns(patterns, runRerunFlag)
+}
+
+func runRerun(cmd *cobra.Command, args []string) error {
+	var patterns []string
+	type jsonTest struct {
+		Name   string                `json:"name"`
+		Result testresult.TestResult `json:"result"`
+		Output string                `json:"output"`
+	}
+	type jsonSuiteResult struct {
+		Tests  []jsonTest            `json:"tests"`
+		Result testresult.TestResult `json:"result"`
+
+		// Context variables
+		Platform string `json:"platform"`
+		Version  string `json:"version"`
+	}
```

Comment:
I think we should be able to delete this now. `DeserialiseReport` returns the internally-defined type, so this shouldn't be used at all.

Reply:
Ah shoot I meant to remove that :sweat_smile: